### ISSUE TITLE
iOS 15.7.2 doesn't work

### ISSUE THREAD

**Issue by PeakGymnast** [ginsudev/WDBFontOverwrite#8]
So Apple has fixed this security bug,right?

**Comment by zhuowei** [comment]
Yes. Unfortunately, this won't be able to work with your device.

I've heard that iOS 15.6 RC is still being signed ([1](https://www.reddit.com/r/jailbreak/comments/yl2knt/tipios_156rc1_is_still_signed_downgrade_now/), [2](https://www.reddit.com/r/ios/comments/zee6y5/is_ios_156_rc_still_being_signed/), [3](https://www.iphonetricks.org/downgrade-ios-16-2-to-15-without-losing-data-ios-15-6-rc/)), so if you really want to downgrade just for this, that might be an option.

**Comment by PeakGymnast** [comment]
> I've heard that iOS 15.6 RC is still being signed ([1](https://www.reddit.com/r/jailbreak/comments/yl2knt/tipios_156rc1_is_still_signed_downgrade_now/), [2](https://www.reddit.com/r/ios/comments/zee6y5/is_ios_156_rc_still_being_signed/), [3](https://www.iphonetricks.org/downgrade-ios-16-2-to-15-without-losing-data-ios-15-6-rc/)), so if you really want to downgrade just for this, that might be an option.

OK，thanks for your suggestion . Apple has revoked iOS 15.6.1 and later hotfix of iOS 15.7,but Delayed upgrade can be applied to downgrade iOS system ,i will downgrade later.
